common/persistence/nosql/nosqlplugin/cassandra/gocql/session_test.go

```diff
@@ -68,4 +70,14 @@ func TestSessionEmitsMetricOnRefreshThrottle(t *testing.T) {
 	metricsHandler.EXPECT().Counter(metrics.CassandraSessionRefreshFailures.Name()).Return(metrics.NoopCounterMetricFunc)
 
 	s.refresh()
+	controller.Finish()
+}
+
+func TestPanicCapture(t *testing.T) {
+	_, err := initSession(log.NewNoopLogger(), func() (*gocql.ClusterConfig, error) {
+		return &gocql.ClusterConfig{Hosts: []string{"0.0.0.0"}}, nil
+	}, metrics.NoopMetricsHandler)
+
+	assert.Error(t, err)
+	assert.Contains(t, err.Error(), "panic:")
```

Comment:
If gocql ever fix this panic issue then this test will break. I think we can just do `assert.NotPanics()`. or just remove this line, because test will fail anyway if the panic is not captured.

Reply:
But I want to verify that I captured the panic.

Reply:
This panic issue in gocql was reported in 2019 and has not been fixed yet. 
I think when gocql fixed this panic, then we could adjust this test.